src/main/java/com/coreos/jetcd/cluster/MemberAddResponse.java

```diff
@@ -34,7 +30,17 @@ public Member getMember() {
    * returns a list of all members after adding the new member.
    */
   public List<Member> getMembers() {
+    List<Member> members = membersRef.get();
+    if (members == null) {
+      synchronized (membersRef) {
+        members = membersRef.get();
+        if (members == null) {
+          members = Util.toMembers(getResponse().getMembersList());
+          membersRef.lazySet(members);
+        }
+      }
+    }
+
```

Comment:
The atomicref used here is to cache the converted result from api.MemberList to Cluster.MemberList. so that subsequent calling of getMembers return the cached value. can you elaborate why `AtomicReference won't be required`?